content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,89 @@ Spring Boot starter, see
 
 ## Configuration
 
+### Bill of Materials
+
+Import the `opentelemetry-bom` and `opentelemetry-instrumentation-bom-alpha`
+BOMs before using the OpenTelemetry starter.
```

Comment:
Some people might not know what a BOM is and why they need it. As of today this is the first reference to this topic, so adding a few words on what it is and why you want it/need it, would be helpful to the reader.

Reply:
@svrnm A BOM is something usual in the Java world. [This Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/gradle-plugin/reference/htmlsingle/) just use "bom". I have added explanations about the interest of using BOMs with the OTel starter:

    The `opentelemetry-bom` BOM will allow you to align the OpenTelemetry versions
    of the OpenTelemetry dependencies and transitive dependencies.
    
    With the `opentelemetry-instrumentation-bom-alpha` BOM, you won't have to
    declare the dependency versions for additional instrumentations.

Reply:
@svrnm I have added a link to the BOM definition: 3783e53408a1359c2681502ac8f7dfe17db34b63 Do you think that it's a good trade off to not re-explain a classical notion in the Java world but to offer the possibility to quickly have the definition for some junior Java developers?